### DISCUSSION TITLE
How can I synchronize the output of Idea with the TID of Skywalking?

### DISCUSSION THREAD

**Discussion by LuanShiZhiJian** [apache/skywalking#11503] [Q&A]
![image](https://github.com/apache/skywalking/assets/41558924/63ba775b-72ce-4003-b05e-f9308154cc95)
![image](https://github.com/apache/skywalking/assets/41558924/ce518036-8e74-42f9-98f2-ddac5c180014)

xml-FILE from https://github.com/apache/skywalking/blob/0f8c67822d9e07cc446e66579c57cfcb2e035297/test/e2e-v2/java-test-service/e2e-service-provider/src/main/resources/logback.xml

I hope that the Tid of both should be consistent.

how to check it? Thanks

**Comment by wu-sheng** [comment]
Why do you mean `synchronize` and `consistent`? Which subjects are you talking about?

**Reply by LuanShiZhiJian** [reply]
What is the meaning of the different TID in IDEA and Skywalking?

**Reply by wu-sheng** [reply]
I can't follow your meaning. It depends on which trace context is related to your logs. 
Your question is too general to answer.

If you want help, I would recommend you at least understand what is distributed tracing and tracing context. Also, you are better to know there are various tracing and threads in the whole JVM, it is impossible to say the connection.

**Reply by wu-sheng** [reply]
You own the codes, and you installed the agent, you should be able to find out the connection through debug, rather than this general question.
